builtin/logical/ssh/path_sign.go

```diff
@@ -48,6 +49,14 @@ func pathSign(b *backend) *framework.Path {
 				Type:        framework.TypeString,
 				Description: `The desired role with configuration for this request.`,
 			},
+			"valid_after": &framework.FieldSchema{
+				Type:        framework.TypeInt,
```

Comment:
I see why you've picked a type int to receive the epoch seconds. It matches the spec you've linked.

We're very big on usability for the Vault API, and we also tend to heavily standardize our fields. I wasn't able to find any precedents for API fields in Vault that represent a specific point in time. That means these two fields might inadvertently create a precedent for how the Vault API consumes date fields! :-) 

I had always imagined that we would tend to get and give date fields that would be in the [ISO-8601](https://en.wikipedia.org/wiki/ISO_8601) format due to both their _human readability_ and their interoperability. I'm pretty sure `2019-07-08T16:39:51Z` would be a forward-compatible date-time format should we ever want to create a framework field for that. 

Would you be willing to switch this over to being a string that only accepts dates and times in that format?

Reply:
Funnily enough, I initially started implementing this from the top down (Vault API first) as ISO-8601 strings, and when I finally got down to the struct passed to the OpenSSH signing lib saw that it was unix epoch's, I just rolled it all back and used int's to represent an epoch.

I'd be find with switching it to a text based ISO 8601 string, if you could point me to an example of the way you would expect to parse ISO 8601 strings with Go (Go is not my primary development language, so I don't know what the de-facto standard is for standards based time parsing)

Reply:
Sure! Thank you! Go does it in an unusual way. [Here](https://golang.org/pkg/time/#Parse) is the "official" tome, and in a nutshell, it would be:
```
whateverVar, err := time.Parse("2006-01-02T15:04:05Z", userValueString)
```

You might want to double-check me, but I believe that's it.